doc/contributors/design-proposals/pull-bundle-on-a-cluster.md

```diff
@@ -0,0 +1,80 @@
+# Pull Bundle on a Cluster
```

Comment:
Could you include why the proposed change is being made?

Reply:
I added a little bit, but maybe more is necessary.